From: Braving Authors <support@braving.org>
Date: Fri, 5 Sep 2026 10:16:00 +0000
Subject: [PATCH] [build] Set Braving minimal feature defaults for Android

Disable non-essential components (Crypto Wallet, VPN, Leo AI Chat, News,
Rewards, Ads) at build time in android_defaults.gni.
---
 build/args/android_defaults.gni | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/build/args/android_defaults.gni b/build/args/android_defaults.gni
index bc218ce..a31e84d 100644
--- a/build/args/android_defaults.gni
+++ b/build/args/android_defaults.gni
@@ -15,3 +15,11 @@ enable_mdns = true
 
 # We want it to be enabled for all configurations
 disable_android_lint = false
+
+# Braving minimal defaults
+enable_brave_wallet = false
+enable_brave_vpn = false
+enable_ai_chat = false
+enable_brave_news = false
+enable_brave_rewards = false
+enable_brave_ads = false
